app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroup.controller.spec.js

```diff
@@ -12,14 +12,14 @@
    describe('filtering', function() {
 
      // Initialize the controller and a mock scope
-     beforeEach(window.inject(function ($controller, $rootScope, $q, accountService, securityGroupReader, modalWizardService,
+     beforeEach(window.inject(function ($controller, $rootScope, $q, accountService, securityGroupReader, v2modalWizardService,
                                  taskMonitorService, azureSecurityGroupWriter) {
 
        this.$scope = $rootScope.$new();
        this.$q = $q;
        this.accountService = accountService;
        this.securityGroupReader = securityGroupReader;
-       this.modalWizardService = modalWizardService;
+       this.modalWizardService = v2modalWizardService;
```

Comment:
AFAICT, this service isn't used or injected by the `azureCreateSecurityGroupCtrl` - if you remove this line and the parameter in the `inject` call above, do these tests still pass?